src/engraving/libmscore/chordline.h

```diff
@@ -92,6 +93,9 @@ class ChordLine final : public EngravingItem
 
     bool isToTheLeft() const { return _chordLineType == ChordLineType::PLOP || _chordLineType == ChordLineType::SCOOP; }
     bool isBelow() const { return _chordLineType == ChordLineType::SCOOP || _chordLineType == ChordLineType::FALL; }
+
+    void setNote(Note* note) { _note = note; }
```

Comment:
that's a bit strange if chordline is attached to one of the notes within a chord. I'd rather expect a chord as a base point for a chord-line

Reply:
well, we'll fix this up later on

Reply:
Yes that is the problem. IMO, this class should have been called `NoteLine` and made it with a `Note` parent, not `Chord` parent, because these type of lines are conceptually similar to glissandos, so they should be attached to _notes_. But changing the whole class implementation is not an option right now, so I did this temporary fix, until we do it properly.